partner_firstname/views/res_partner.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+Odoo, Open Source Management Solution
+Author: Nicolas Bessi. Copyright Camptocamp SA
+Copyright (C)
+    2014:       Agile Business Group (<http://www.agilebg.com>)
+    2015:       Grupo ESOC <www.grupoesoc.es>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<openerp>
+<data>
+<record id="partner_simple_form" model="ir.ui.view">
```

Comment:
@bwrsandman I think @Olivier-LAURENT's point is that other (probably customer-specific) modules depending on partner_firstname break if you do this. Given that this is a major overhaul, those modules probably will need to be adapted anyway, but I also don't see the benefit in this id change.
Depending on how the depending modules' overrides are crafted, they might survive survive the changes and only depend on this form to be sure to be applied after this did its work. And then, it indeed causes unnecessary work.